kmelia/kmelia-war/src/main/java/org/silverpeas/components/kmelia/control/KmeliaSessionController.java

```diff
@@ -1069,40 +1066,47 @@ private synchronized void orderPubs(int sortType) {
     List<KmeliaPublication> publicationsToSort = new ArrayList<>(publications);
 
     int sort = sortType;
-    if (isManualSortingUsed(publicationsToSort) && sort == -1) {
+    boolean manualSortingUsed = isManualSortingUsed(publicationsToSort);
+
+    if (manualSortingUsed && !getSort().isExplicitSort()) {
       // display publications according to manual order defined by admin
       sort = 99;
-    } else if (sort == -1) {
+    } else if (getSortValue() == Sort.SORT_MANUAL) {
       // display publications according to default sort defined on application level or instance
       // level
       sort = getDefaultSortValue();
     }
 
+    // store current sort
```

Comment:
I don't understand what is the manual order here. The code suggests it is the order defined by the admin, that is to say either in the component settings or in the component instance parameterization. But in another way, the word "manual" suggests that the order was done by a user (with the right to reorder the publication listing of a topic). 
What is exactly manual sorting?

Reply:
Manual sort is manual sort. Default sort is the one defined by the admin, that is to say either in the component settings or in the component instance parameterization.